docs/input/docs/getting-started/principles.md

```diff
@@ -0,0 +1,16 @@
+---
+Order: 20
+Title: Technical core principles
+Description: Technical core principles of BBT.Maybe
+---
+
+C# does not (yet) include a language construct for null safe treatment of optional reference-type variables.
+Therefore each and every method argument, return value, class member, property and local variable is a challenge.
+A possible solution is implementing a null condition for each reference call with an if clause.
```

Comment:
See comment above. Describing the challenge will help the flow here, as the solution gets a relation to the challenge.

Reply:
Done.